main/guides/orchestration/chainhub.md

```diff
@@ -0,0 +1,118 @@
+# ChainHub
+
+The [ChainHub API](https://github.com/Agoric/agoric-sdk/blob/859d8c0d151ff6f686583db1eaf72efb89cc7648/packages/orchestration/src/exos/chain-hub.js#L99) is responsible managing chain and IBC connection information. It facilitates the registration and retrieval of chain and connection data.
+
+```js
+const zone = makeDurableZone(baggage);
+const { agoricNames } = remotePowers;
+const chainHub = makeChainHub(agoricNames, zone);
+```
+
+The `makeChainHub` function accepts a `Remote<NameHub>` reference (`agoricNames`) and an optional `Zone` for managing data durability. The `makeChainHub` fuction creates a new `ChainHub` instance either in the specified zone or in the heap if no zone is provided. The resulting object is an Exo singleton, which means it has no previous state. Its state consists only of a cache of queries to `agoricNames` and the information provided in registration calls.
+
+The `ChainHub` objects maintains two `MapStores`:
+
+- `chainInfos`: For storing `CosmosChainInfo` objects.
+- `connectionInfos`: For storing `IBCConnectionInfo` objects.
+
+These `MapStores` are not exposed directly. They are abstracted and used internally by the methods provided by the ChainHub.
+
+# ChainHub Interface
+
+The core functionality is encapsulated within the `makeChainHub` function, which sets up a new `ChainHub` in the given zone. The `ChainHub` is responsible for:
+
+## **chainHub.registerChain(name, chainInfo)**
```

Comment:
Either remove `chainHub` in all APIs or add to all of them. 
I think for cleaner presentation you may just delete `chainHub` in all. 

Also, because this is a heading, you may also delete the parameters. But I haven't seen at other pages to confirm what is the convention here.

Reply:
Adding `chainHub` with all APIs. Followed conventions [from here](https://rs-chainhub-documentation.documentation-7tp.pages.dev/reference/ertp-api/issuer.html#issuer-object)